src/app/app.component.html

```diff
@@ -1,7 +1,11 @@
-<alm-app-header></alm-app-header>
-
-<main class="app">
-  <router-outlet></router-outlet>
+<header class="padding-bottom-10" >
```

Comment:
I don't think there is anything that says alm-app-header needs a div at the top level. It could be a header tag. The idea is that the header is self-contained.